Product Approval Process + Merchant Context (#763)
spryker/spryker-docs: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md

## Prerequisites

To start working with marketplace concrete products, go to the **Merchant Portal** > **Products**.  Hover over the three dots next to the abstract product for which you manage a concrete product and click **Manage Product** or just click the line. This takes you to the *[Product name]* drawer. Navigate to the *Concrete Products* tab.

> This article contains reference information. Make sure to review it before you start, or just look up the necessary information as you go through the process.

Reviewer: same as above

## Filtering and sorting concrete products

> The following drop-down menus can be used to filter the marketplace concrete products on the *Concrete Products* page:

Reviewer: ```suggestion
You can use the following drop-down menus to filter the marketplace concrete products on the *Concrete Products* page:
```

## Activating and deactivating a concrete product

To activate a marketplace concrete product:

1. On the *Concrete Products* page, next to the concrete product you want to activate, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page.
2. In the *Status* pane, select *Concrete Product is online* to make the concrete product online.
3. Click **Save**.

> To deactivate the product, deselect *Concrete Product is online* to make the concrete product offline.

Reviewer: Please revert to "clear": https://developers.google.com/style/word-list#letter-d
